doc/pages/architecture/030-interface.md

```diff
@@ -74,6 +74,10 @@ The following types are supported:
 * `doublearray`, `integerarray`, `booleanarray`, `longintegerarray`, `stringarray`, `binaryblobarray`, `datetimearray`: A list of values, represented as a JSON Array. Arrays can have up to 32768 items, must be shorter than 1MiB, and each item must be shorter than 64KiB. In particular, text fields must be shorter than 32000 characters.
 
 ### Limitations
+A valid interface name consists of a Reverse Domain Name containing alphanumeric characters, hyphens and dots. By design, both the top level domain and last domain component can not contain hyphens.
```

Comment:
This is "Limitations" under the "Mappings" section. It's probably better to add another "Limitations" section under "Format" above (that already talks about the name format),